ccmlib/cmds/cluster_cmds.py

```diff
@@ -693,8 +693,6 @@ def get_parser(self):
         parser = self._get_default_parser(usage, self.description())
         parser.add_option('--no-hh', '--no-hinted-handoff', action="store_false",
                           dest="hinted_handoff", default=True, help="Disable hinted handoff")
-        parser.add_option('--batch-cl', '--batch-commit-log', action="store_true",
```

Comment:
I'm wary of breaking this API -- we have no idea who uses this option. I guess our strategies are to either reimplement it on top of the new implementation, or to remove it and only reimplement if someone complains. Thoughts @ptnapoleon?